src/components/ClassCard.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+    <v-card
+        class="mx-auto"
+        id="card"
+        elevation="24"
+    >
+        <v-img
+        :src="image"
+        height="200px"
+        ></v-img>
+
+        <v-card-title>
+        {{ title }}
+        </v-card-title>
+
+        <v-card-subtitle>
+        {{ summary }}
+        </v-card-subtitle>
+
+        <v-card-actions>
+        <v-btn
+            id="button-reserve"
+            text
+        >
+            RESERVA
+        </v-btn>
+
+        <v-spacer></v-spacer>
+
+        <v-btn
+            icon
+            @click="show = !show"
+        >
+            <v-icon>{{ show ? 'mdi-chevron-up' : 'mdi-chevron-down' }}</v-icon>
+        </v-btn>
+        </v-card-actions>
+
+        <v-expand-transition>
+        <div v-show="show">
+            <v-divider></v-divider>
+
+            <v-card-text>
+            {{ content }}
+            </v-card-text>
+        </div>
+        </v-expand-transition>
+    </v-card>
+</template>
+
+<style lang="scss" scoped>
+    div.mx-auto.v-card {
+        width: 90%;
+        max-width: 325px;
+        margin: 10px;
+    }
+    #card {
+        height: 100%;
+    }
+    #card,
+    #card div,
+    .v-btn.v-btn--flat.v-btn--icon
+    {
+        background-color: $dark-theme-surface;
+        color: $main-font-color;
+        font-family: $section-body-font;
+        font-size: 12px;
+        @media (min-width: $breakpoint-desktop) {
+            font-size: 18px;
+        }
+    }
+    #button-reserve {
+        color: $main-links-color;
+        font-family: $footer-title-font;
```

Comment:
I think this element should have its own variables, so the names don't get mixed up